modules/helper.py

```diff
@@ -404,14 +404,13 @@ def autoRotate(ifile):
             result = debug.subprocess_check_output(['/usr/bin/jpegexiforient', ifile])  # , stderr=void)
         if result:
             orient = int(result)-1
-            if orient < 0 or orient >= len(parameters):
-                logging.info('Orientation was %d, not transforming it', orient)
+            if not 0 < orient < len(parameters):
+                logging.debug('Orientation was %d, not transforming it', orient)
                 return ifile
             cmd = [helper.TOOL_ROTATE]
             cmd.extend(parameters[orient].split())
             cmd.extend(['-outfile', ofile, ifile])
-            with open(os.devnull, 'wb') as void:
-                result = debug.subprocess_check_call(cmd, stderr=void)
+            result = debug.subprocess_call(cmd, stderr=subprocess.STDOUT)
```

Comment:
Why change this to `STDOUT` ?

Reply:
I was seeing the `subprocess_check_call` command failing, and that was crashing the slideshow thread.  I "think" it was because the result needed not to be a string.   `subprocess_call` does not have the `.decode('utf-8')` from its debug version.  I was trying various alternatives, including the change to STDOUT in case there was also a type matching problem between the stderr output and opening /dev/null as binary.  I left it that way in the end b/c I thought it might less expensive.  If you like, I can test it with just the `subprocess_call` change and revert if it still works.

Reply:
Hmm... I think I need to move the subprocess calls away from the debug file. Should probably be in the helper, and it might be worth to rewrite it to capture the output and in the case of failure, dump the output into the log and if not, swallow the output.

Reply:
I'm ok with keeping it as-is for now. I actually think you can skip the `stderr=` part and then it will default to stderr by default.